### ISSUE TITLE
Add semantics for CupertinoExpansionTile

### ISSUE THREAD

**Issue by victorsanni** [flutter/flutter#173446]
### Use case

Similar to ExpansionTile, CupertinoExpansionTile should announce its expansion state for assistive technologies (voiceover).

### Proposal

Announce expansion state when expanded/collapsed.

**Comment by chunhtai** [comment]
Maybe we should move the semantics along with the localized label to Expandable widget in widget layer.
